CMakeLists.txt

```diff
@@ -74,7 +74,7 @@ if(NOT TARGET UrlLib AND JSRUNTIMEHOST_POLYFILL_XMLHTTPREQUEST)
     set_property(TARGET UrlLib PROPERTY FOLDER Dependencies)
 endif()
 
-if(JSRUNTIMEHOST_CORE_APPRUNTIME_V8_INSPECTOR)
+if(NAPI_JAVASCRIPT_ENGINE STREQUAL "V8" AND JSRUNTIMEHOST_CORE_APPRUNTIME_V8_INSPECTOR)
```

Comment:
Why did we need this additional check?

Reply:
`JSRUNTIMEHOST_CORE_APPRUNTIME_V8_INSPECTOR` is set to true by default even if the javascript engine is not V8. Without this check, it will try to compile the v8inspector even for non-V8 engines.

Reply:
Production code should explicitly set this to false to prevent debugging and reduce code size.